app/src/main/java/org/jellyfin/androidtv/ui/playback/PlaybackController.java

```diff
@@ -639,14 +639,20 @@ private void startItem(BaseItemDto item, long position, StreamInfo response) {
             mVideoManager.setVideoPath(response.getMediaUrl());
         }
 
-        //wait a beat before attempting to start so the player surface is fully initialized and video is ready
-        mHandler.postDelayed(new Runnable() {
-            @Override
-            public void run() {
-                if (mVideoManager != null)
-                    mVideoManager.start();
+        //set video start delay
+        long videoStartDelay = userPreferences.getValue().get(UserPreferences.Companion.getVideoStartDelay());
+        if (mVideoManager != null) {
```

Comment:
This null cehck should only happen in the delayed function as we need to verify the manager is not disposed. We already know it is not null when calling it directly.

Reply:
I have updated the null check for only when a delay has been set.